docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+- `MANAGE_WITHDRAWAL_CREDENTIALS_ROLE`
+    - None
+- `STAKING_MODULE_PAUSE_ROLE`
+    - DepositSecurityModule
+- `STAKING_MODULE_RESUME_ROLE`
+    - DepositSecurityModule
+- `STAKING_MODULE_MANAGE_ROLE`
+    - None
+- `REPORT_EXITED_VALIDATORS_ROLE`
+    - AccountingOracle
+- `UNSAFE_SET_EXITED_VALIDATORS_ROLE`
+    - None
+- `REPORT_REWARDS_MINTED_ROLE`
+    - Lido
+
+### **HashConsensus for AccountingOracle**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE`
+    - Before vote start:
+	    - Template
+    - After enactment:
+	    - Agent
+- `MANAGE_MEMBERS_AND_QUORUM_ROLE`
+    - None
+- `DISABLE_CONSENSUS_ROLE`
+    - None
+- `MANAGE_FRAME_CONFIG_ROLE`
+    - None
+- `MANAGE_FAST_LANE_CONFIG_ROLE`
+    - None
+- `MANAGE_REPORT_PROCESSOR_ROLE`
+    - None
+- `address[] _memberAddresses`
+	- Before vote start:
+	    - None
+	- After enactment:
+	    - Current LidoOracle Committee
+	       See https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F28
+
+### **DepositSecurityModule**
+*Plain Owner, No Proxy*
+
+- owner
+    - Before vote start:
+	    - Template
+    - After enactment:
+	    - Agent
+- guardians
+	- Before vote start:
+	    - None
+	- After enactment:
+	    - Current DSM guardians committee
+	       See https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F8
+
+### **HashConsensus for ValidatorExitBusOracle**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE`
+    - Before vote start:
+	    - Template
+    - After enactment:
+	    - Agent
+- `MANAGE_MEMBERS_AND_QUORUM_ROLE`
+    - None
+- `DISABLE_CONSENSUS_ROLE`
+    - None
+- `MANAGE_FRAME_CONFIG_ROLE`
+    - None
+- `MANAGE_FAST_LANE_CONFIG_ROLE`
+    - None
+- `MANAGE_REPORT_PROCESSOR_ROLE`
+    - None
+- `address[] _memberAddresses`
+	- Before vote start:
+		- None
+	- After enactment:
+		- Current LidoOracle Committee
+		  See https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F28
+
+### **OracleDaemonConfig**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE`
+    - Agent
+- `CONFIG_MANAGER_ROLE`
+    - None
+
+### **OracleReportSanityChecker**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE` (set in constructor)
+    - Agent
+- `ALL_LIMITS_MANAGER_ROLE`
+- `CHURN_VALIDATORS_PER_DAY_LIMIT_MANAGER_ROLE`
+- `ONE_OFF_CL_BALANCE_DECREASE_LIMIT_MANAGER_ROLE`
+- `ANNUAL_BALANCE_INCREASE_LIMIT_MANAGER_ROLE`
+- `SHARE_RATE_DEVIATION_LIMIT_MANAGER_ROLE`
+- `MAX_VALIDATOR_EXIT_REQUESTS_PER_REPORT_ROLE`
+- `MAX_ACCOUNTING_EXTRA_DATA_LIST_ITEMS_COUNT_ROLE`
+- `MAX_NODE_OPERATORS_PER_EXTRA_DATA_ITEM_COUNT_ROLE`
+- `REQUEST_TIMESTAMP_MARGIN_MANAGER_ROLE`
+- `MAX_POSITIVE_TOKEN_REBASE_MANAGER_ROLE`
+    - None for all roles listed above
+
+### **ValidatorExitBusOracle**
+*OZ, Proxy*
+
+- PROXY ADMIN
+    - Before vote start:
+	    - Template
+    - After enactment:
+	    - Agent
+- `DEFAULT_ADMIN_ROLE`
+    - Before vote start:
+	    - None
+    - After enactment:
+	    - Agent
+- `PAUSE_ROLE` (in initializer)
```

Comment:
Missed the meaning of 'in initializer' and 'in initialize' remarks, and looks like they are unique for this contract.

Reply:
just have removed, ok now?

Reply:
yep